src/tictactoe_clojure/game.clj

```diff
@@ -0,0 +1,32 @@
+(ns tictactoe-clojure.game
+  (:require [tictactoe-clojure.board :as board])	
+  (:require [tictactoe-clojure.human-player :as human-player])
+  (:require [tictactoe-clojure.computer-player :as computer-player])
+  (:require [tictactoe-clojure.console :as console])	
+  (:require [tictactoe-clojure.rules :as rules]))
+
+(defn take-two-moves
+  [board current-turn]
+    (let [board board]
+      (console/display (str current-turn "'s turn"))
+      (if (= current-turn "O")
+        (board/move board (human-player/human-move board) "O") 
+        (board/move board (computer-player/optimal-move board 9) "X"))))
+
+(defn show-result
+  [played-board]
+    (if (rules/draw? played-board)
```

Comment:
Let's think about shell vs core discussion here again. Could this conditional be somewhere else?